### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t2077

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: css/embed.less [hunk 1]
```diff
@@ -210,27 +230,35 @@ body {
...
+#post-roll {
+
   .post-roll-description {
     color: #676867;
+    font-size: 20px;
   }
 
   .embed-nav > li > a {
-    background: rgba(0,0,0,0.3) url(../resources/controls/controls_icons.png) no-repeat;
+    background: rgba( 0, 0, 0, 0.3 ) url(../resources/controls/controls_icons.png) no-repeat;
```
Comment (secretrobotron) [inline css/embed.less]: I saw you put quotes around a urls above (https://github.com/mozilla/butter/pull/895/files#L2R118). We should probably keep things consistent.

File: css/embed.less [hunk 2]
```diff
@@ -304,84 +316,172 @@ body {
...
-  padding: 10px;
-  border: none;
-  float: left;
-  border-radius: 2px;
-  &:focus {
-    outline: none;
-    box-shadow: 0 0 0 1px #2BB673;
+  .share-options textarea {
+    height: 70px;
+    font-family: menlo, monospace ;
```
Comment (secretrobotron) [inline css/embed.less]: Extra space at the end here.

File: css/embed.less [hunk 3]
```diff
@@ -304,84 +316,172 @@ body {
...
-}
+    a:hover {
+      background:#1EB771;
+    }
 
-.share-buttons > li > a {
-  padding: 3px;
-}
+    a.current {
+      display:block;
```
Comment (secretrobotron) [inline css/embed.less]: Extra space please.

File: css/embed.less [hunk 4]
```diff
@@ -304,84 +316,172 @@ body {
...
+      background: url( "../resources/controls/embed_size_sprite.png" ) no-repeat 90px 15px;
+    }
 
-.share-size {
-  width: 100px;
-  float: left;
-  margin-left: 20px;
-}
+    a.current:hover {
+      display:block;
```
Comment (secretrobotron) [inline css/embed.less]: Extra space please.

File: css/embed.less [hunk 5]
```diff
@@ -394,42 +494,14 @@ body {
...
-      font-weight: 700;
-      line-height: 1em;
-    }
-
-    .mozpopcorn {
-      color: #D53B2A;
-      font-weight: bold;
-      font-size: 16px;
-      margin: 0 0 5px 0;
+      height:255px;
```
Comment (secretrobotron) [inline css/embed.less]: Extra space please.

File: css/embed.less [hunk 6]
```diff
@@ -437,101 +509,161 @@ body {
...
+      margin-top: 60px;
+      width: 100%;
+
+      .size-options {
+        bottom: -35px;
       }
 
+    }
+
+    .share-size  label {
```
Comment (secretrobotron) [inline css/embed.less:569]: Too much space between these.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e153752a6b096b7/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/e153752a6b096b7/output.txt

Total script time: 0.94 mins

Server is running on http://pr895.butyr.org/templates/basic/
Run the Test Suite: http://pr895.butyr.org/test/

**Pull request closed without merging**